packages/babel-plugin-transform-typescript/src/index.js

```diff
@@ -42,17 +42,25 @@ export default function() {
         let allElided = true;
         const importsToRemove: Path<Node>[] = [];
 
-        for (const specifier of path.node.specifiers) {
+        for (let i = 0, len = path.node.specifiers.length; i < len; i++) {
```

Comment:
Nit: Usually engines cache the length, so the `len` variable isn't needed

Reply:
Thanks, I'll fix it